core/services/offchainreporting/transmitter.go

```diff
@@ -0,0 +1,37 @@
+package offchainreporting
+
+import (
+	"context"
+	"database/sql"
+
+	gethCommon "github.com/ethereum/go-ethereum/common"
+	"github.com/pkg/errors"
+)
+
+type transmitter struct {
+	db          *sql.DB
+	fromAddress gethCommon.Address
+	gasLimit    uint64
+}
+
+// NewTransmitter creates a new eth transmitter
+func NewTransmitter(sqldb *sql.DB, fromAddress gethCommon.Address, gasLimit uint64) Transmitter {
+	return &transmitter{
+		db:          sqldb,
+		fromAddress: fromAddress,
+		gasLimit:    gasLimit,
+	}
+}
+
+func (t *transmitter) CreateEthTransaction(ctx context.Context, toAddress gethCommon.Address, payload []byte) error {
+	_, err := t.db.ExecContext(ctx, `
+INSERT INTO eth_txes (from_address, to_address, encoded_payload, value, gas_limit, state, created_at)
```

Comment:
TODO: Move to database subscription/trigger otherwise we have to wait for polling

Reply:
Isn't there already a method somewhere that does this exact query?  Seems like deduplicating would be best so that if we ever change the eth_txes table, we don't forget to update one of the locations where this query happens.